openlibrary/templates/type/author/edit.html

```diff
@@ -4,6 +4,8 @@
 
 $putctx("robots", "noindex,nofollow")
 
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
I think we may want to do this without importing font-awesome, but otherwise the link looks fine.